lib/src/components/NxCodeSnippet/NxCodeSnippet.scss

```diff
@@ -42,6 +42,7 @@
   .nx-text-input {
     grid-area: textarea;
     width: 100%;
+    overflow-x: hidden;
```

Comment:
This deserves a code comment explaining what it's for. Calling out that it's firefox-specific and referencing either the stackoverflow link or the actual firefox bug report link.